content/sensu-go/5.15/reference/etcdreplicators.md

```diff
@@ -61,7 +61,7 @@ example      | {{< highlight shell >}}api_version: federation/v1{{< /highlight >
 
 metadata     |      |
 -------------|------
-description  | Top-level scope that contains the replicator `name`.
+description  | Top-level scope that contains the replicator `name`. Namespace is not supported because EtcdReplicators are cluster-wide resources.
```

Comment:
```suggestion
description  | Top-level scope that contains the replicator `name`. Namespace is not supported in the metadata because EtcdReplicators are cluster-wide resources.
```